Test/DependencyCollector/FunctionalTests/TestApps/AspxCore20/Startup.cs

```diff
@@ -0,0 +1,50 @@
+namespace AspxCore
+{
+    using Microsoft.AspNetCore.Builder;
+    using Microsoft.AspNetCore.Hosting;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.ApplicationInsights.DependencyCollector;
+    using Microsoft.ApplicationInsights.Extensibility;
+
+    public class Startup
+    {
+        public Startup(IHostingEnvironment env)
+        {
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(env.ContentRootPath)
+                .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true);
+            
+            builder.AddEnvironmentVariables();
+            Configuration = builder.Build();
+        }
+
+        public IConfigurationRoot Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // Add framework services.            
+            services.AddMvc();
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
+        {
+            loggerFactory.AddConsole(Configuration.GetSection("Logging"));
+            loggerFactory.AddDebug();
+
+            app.UseMvc();
+
+            var teleConfig = TelemetryConfiguration.Active;
+            teleConfig.TelemetryChannel.DeveloperMode = true;            
+
+            // Fake endpoint.
+            teleConfig.TelemetryChannel.EndpointAddress = "http://localhost:8789/v2/track";
```

Comment:
Minor: Looks hardcoded - do we start this endpoint ourselves, so we can pass a variable here as a port?

Reply:
we start it ourself.. will adjust

Reply:
just realized that no common place to have all constants. its hardcoded to all projects in ai.config or startup.